ibm/resource_ibm_network_gateway_vlan_attachment_test.go

```diff
@@ -11,67 +11,22 @@ import (
 func TestAccIBMNetworkGatewayVlanAtachment_Basic(t *testing.T) {
 
 	hostname1 := fmt.Sprintf("tfuat%s", acctest.RandString(11))
-	hostname2 := fmt.Sprintf("tfuat%s", acctest.RandString(11))
 	gatewayName := fmt.Sprintf("tfuatgw%s", acctest.RandString(12))
 
-	config := "ibm_network_gateway.gw"
-
 	resource.Test(t, resource.TestCase{
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: testAccCheckIBMNetworkGatewayVlanAttachment_basic(gatewayName, hostname1, hostname1),
+				Config: testAccCheckIBMNetworkGatewayVlanAttachment_basic(gatewayName, hostname1),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
 						"ibm_network_gateway_vlan_association.attachment", "bypass", "true"),
-					resource.TestCheckResourceAttr(
-						config, "members.#", "2"),
```

Comment:
Please add the check for number of members

Reply:
@sakshiag Vlan attachment UAT is basically for validating that particular resource.. Gateway UAT has been taking care of validating Gateway resource